### PR TITLE
Pushing Execute Search

### PR DESCRIPTION
**Pull request by alexjoybc**
This PR includes the following proposed changes:
* `ExcuteSearch` contract
* `MassTransit` configuration to push messages

### CODE REVIEW

File: app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SearchApi.Web.Controllers
+{
+    public interface ExecuteSearch
```
Comment (peggy-gov-bc-ca) [inline app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs]: shall we use IExecuteSearch to make it clear that it is an interface?

Comment (alexjoybc) [reply app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs]: Hehe, I kinda agree with you, but in MassTransit the contract don't start with the I reference....

Comment (alexjoybc) [reply app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs]: https://masstransit-project.com/MassTransit/usage/producing-messages.html#sending-via-interfaces

Comment (peggy-gov-bc-ca) [reply app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs]: good to know.

**Pull request merged**